course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+2. Basic R programming, vectors, data frames, loops, conditions, etc.
+3. Before 2nd class: Everyone have Rstudio up and running. Maybe we can also link them to some R intro course and ask them to complete 1 or 2 lectures there per week, which will serve as the underlying material for the quiz? This one is good imo http://tryr.codeschool.com/. Watch Christie's talk https://www.youtube.com/watch?v=TthxI9mlvGE?
+
+### Week 2
+1. Basic R-programming, repetition and continued, scripting.
+2. dplyr intro, finalize R-prep by teaching outstanding items required to use R for the following ecology classes.
+3. Again, link online lectures?
+
+### Week 3
+1. Ecology background knowledge for lab 1
+2. Ecology lab 1 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 4
+1. Ecology background knowledge for lab 2
+2. Ecology lab 2 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 5
+1. Introduce data sets, divide intro groups, think about interesting questions to ask, formulate hypothesis/plan of action.
+    - Potentially do this already the week before and have them take a survey with the data they want to work on and their background experience.
+2. Project work, getting started with their own data in R, tidy data principles, how to clean data, setting up a collaborative git environment with Rstudio (how much do we want to get into git?).
+3. Reflections blog post of what they have learnt so far?
+
+### Week 6
+1. Data visualization in general and in R, ggplot, interactive (since plotly is just one additional line)?
+2. Data visualization continued, project specific.
+3. Online tutorial about plotting.
+
+### Week 7
+1. Stuff important for reproducible workflow, metadata, Rmarkdown, Roxygen?
+2. Project work,
+3. Write a data README?
+
+### Week 8-12
+I have left these open for now, since I expect there will be many things that will be reorganized anyways. Ideally one lecture per week from now on would be project work only. Potential ideas:
+    - A couple of additional ecology lessons?
+    - More info about working together on online collaborative platforms? Both for code and writing.
+    - More about data openness?
+    - Guest lectures?
+    - Preparing a scientific item for academic publishing
+    - Licensing and authorship
+- Let me know your overall thoughts on the draft so far. Maybe it is too compressed, I think it is hard to estimate exactly how long we need to spend on certain topics, and I might have been a bit optimistic.
+
+## Labs and lectures from Martin
+These are the topics from what Martin sent us on Dropbox.
+
+### Structured population dynamics
+- Expressing population changes with matrix algebra
+- Learning how to use R for matrix operations and basic plotting
+- Eigenvalues
+- Elasticity of growth rates
+- Gaussian distribution fitting, mean, standard deviation, probability density function.
+
+### Predator prey population dynamics
+- First order, non-linear, differential equations.
```

Comment:
I feel very confident that I can cover expressing population changes as a matrix, eigenvalues, differential equations, stability analysis, equilibrium. That is literally exactly what I've been doing for the last year. I just found out that I'm an ecologist! I can also probably do fitting and probability density function, etc., but it'll be good to clarify what conventions they use. I'll need to work a bit harder to practice doing all that stuff in R instead of Python.